src/Microsoft.IdentityModel.Tokens/JsonWebKey.cs

```diff
@@ -333,6 +333,32 @@ internal RSAParameters CreateRsaParameters()
             };
         }
 
+#if NET472
+        internal ECParameters CreateEcParameters()
+        {
+            //todo: update log msgs
+            if (string.IsNullOrEmpty(Crv))
+                throw LogHelper.LogExceptionMessage(new ArgumentException(LogHelper.FormatInvariant(LogMessages.IDX10700, LogHelper.MarkAsNonPII(_className), LogHelper.MarkAsNonPII("Crv"))));
+
+            if (string.IsNullOrEmpty(X))
+                throw LogHelper.LogExceptionMessage(new ArgumentException(LogHelper.FormatInvariant(LogMessages.IDX10700, LogHelper.MarkAsNonPII(_className), LogHelper.MarkAsNonPII("X"))));
+
+            if (string.IsNullOrEmpty(Y))
+                throw LogHelper.LogExceptionMessage(new ArgumentException(LogHelper.FormatInvariant(LogMessages.IDX10700, LogHelper.MarkAsNonPII(_className), LogHelper.MarkAsNonPII("Y"))));
```

Comment:
>"Y" [](http://example.com/codeflow?start=181&length=3)

nameof(Y)

Reply:
Removing this piece of code as is no longer used